tests/suite/js.html

```diff
@@ -456,8 +456,7 @@
     bar = /\/{/
     baz = //{//
 
-    // The following is known to be broken
+    // The following is no longer broken \o/
     /function() {/
-        return
-    }
+    return
 </script>
```

Comment:
Can we add a test-case here with a single-character regular expression? Because the current change would break the tokenization of such a thing.

```
english_hello = "hallo".replace(/a/, "e")
```

Reply:
Done, although I have chosen the `/{/` regex as it would break indentation.